addon/components/au-date-picker.hbs

```diff
@@ -1,7 +1,20 @@
-<div class="au-date-picker">
+<div class="au-date-picker" data-test-au-date-picker>
   {{#if @label}}
-    <AuLabel for="date">{{@label}}</AuLabel>
+    <AuLabel for={{this.id}} data-test-au-date-picker-label>{{@label}}</AuLabel>
```

Comment:
I seem to remember there being issues with html attributes without values in glimmer, I think only specific attributes can exist without a value (such as checked, disabled, etc). Unfortunately I can't find the issue immediately.

Reply:
What kind of issues? I can't recall anything like that :smile:.

Reply:
since I can't find it again, I guess we can assume this is resolved :+1: